src/wp-admin/js/customize-controls.js

```diff
@@ -9208,7 +9250,6 @@
 		// Toggle visibility of Header Video notice when active state change.
 		api.control( 'header_video', function( headerVideoControl ) {
 			headerVideoControl.deferred.embedded.done( function() {
-				// eslint-disable-next-line func-style
```

Comment:
`// eslint-disable-next-line func-style`

Leave this comment line in the code to silence `Expected a function declaration` errors / warnings in ESLint.

Reply:
Thanks for the explanation! Now done.